conf/modules/ins_ext_pose.xml

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="ins_ext_pose" dir="ins">
+  <doc>
+    <description>
+    extended kalman filter made by robin.
```

Comment:
Batman might be frustrated not to cited here. Any additional link or info about this filter or this particular implementation ?